src/python/tests/core/bot/tasks/corpus_pruning_task_test.py

```diff
@@ -94,7 +94,9 @@ def mocked_unpack_seed_corpus_if_needed(*args, **kwargs):
     self.corpus_dir = os.path.join(self.corpus_bucket, 'corpus')
     self.quarantine_dir = os.path.join(self.corpus_bucket, 'quarantine')
     self.shared_corpus_dir = os.path.join(self.corpus_bucket, 'shared')
+    self.fuchsia_corpus_dir = os.path.join(self.corpus_bucket, 'fuchsia')
```

Comment:
Put these fuchsia specific ones into the fuchsia test subclass.

Reply:
done!